dtbase/tests/test_model_arima.py

```diff
@@ -41,12 +47,13 @@ def test_arima_get_temperature(
 
 @pytest.mark.skipif(not DOCKER_RUNNING, reason="requires docker")
 def test_arima_get_humidity(conn_backend: None, session: Session) -> None:
-    insert_trh_readings(session)
+    insert_trh_readings(session, add_noise=False)
```

Comment:
Is the `add_noise=False` still needed now that the seed is fixed? Same question for several calls below.

Reply:
Correct. I'm still going to keep it as an option in the function because it could be useful but I won't call it here.

Reply:
By correct, I mean add_noise isn't need I don't think